q2_diversity/_beta/_pipeline.py

```diff
@@ -0,0 +1,68 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2020, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from q2_diversity_lib.beta import METRICS
+from q2_diversity_lib import translate_metric_name
+
+all_phylo_metrics = METRICS['PHYLO']['IMPL'] | METRICS['PHYLO']['UNIMPL']
+all_nonphylo_metrics = METRICS['NONPHYLO']['IMPL'] \
+                       | METRICS['NONPHYLO']['UNIMPL']
+metric_name_translations = METRICS['METRIC_NAME_TRANSLATIONS']
+
+
+def all_metrics():
+    return all_phylo_metrics | all_nonphylo_metrics
+
+
+def beta_phylogenetic(ctx, table, phylogeny,
+                      metric, threads=1,
+                      variance_adjusted=False,
+                      alpha=None,
+                      bypass_tips=False):
+    if metric not in all_phylo_metrics:
+        raise ValueError("Unknown metric: %s" % metric)
```

Comment:
Let's drop this guard - QIIME 2 protects us from this case, and we don't support calling via python, directly. These "duplicated guards" are leftovers from our early days with developing QIIME 2 plugins.

Reply:
Removed from `alpha`, `alpha_phylo`, `beta`, `beta_phylo`.